### ISSUE TITLE
ux_device_class_cdc_acm_read read up arrow key as 0x0D

### ISSUE THREAD

**Issue by calism** [azure-rtos/usbx#104]
Hello All,

- I use the function ux_device_class_cdc_acm_read to read the inputs from Terminal (in this case Tera Term).
- If I press  up arrow key, the function ux_device_class_cdc_acm_read writes into the buffer the value 0x0D which is the Carriage Return.

So, I assume there is an issue the function ux_device_class_cdc_acm_read

**Comment by xiaocq2001** [comment]
I don't think so, up arrow key is not interpreted as normal character, it could be sequence of keys. Maybe you can check trace of the USB traffic, I think it gets Carriage Return because host really send it in its key sequence.
